tools/cloud/demo.sh

```diff
@@ -527,6 +573,18 @@ aws iotfleetwise update-campaign \
     --name ${NAME}-campaign \
     --action APPROVE | jq -r .arn
 
+echo "Suspending campaign..."
```

Comment:
why suspend and resume after approve? It this pure for demo purpose? Maybe add a comment to illustrate the intention here

Also as there's no delay between each command. Could we run into timing issue such as API returned but the campaign status is still propagating in the backend?

Reply:
Yes, this is purely for functionality demo, I will add the comments here. Good catch, it may happen, though in backend, it's light-weight status mapping. I will add a short pause. Thanks!